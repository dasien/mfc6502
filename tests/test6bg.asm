LDX #$01   ;X is $01
LDA #$aa   ;A is $aa
STA $a0,X ;Store the value of A at memory location $a1
ADC #01
INX        ;Increment X
STA $a0,X ;Store the value of A at memory location $a2
