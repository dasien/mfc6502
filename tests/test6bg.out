A2 01
A9 AA
95 A0
E8 
95 A0
